engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -282,6 +282,13 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
     for {
       _ <- publishLabelsToMetadata(id, labels)
       zippedImportResolver <- zippedResolverCheck
+      // Debugging #4117 - make sure we are creating an importable directory out of the zip (AEN 2018-11-14)
+      _ = zippedImportResolver match {
+        case Some(dr: DirectoryResolver) =>
```

Comment:
Why the ascription here? It appears to be redundant. If it turns out not to be redundant then there's a non-exhaustive match which would be a bigger problem. 🙂